src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/AvoidStarImportCheck.java

```diff
@@ -74,6 +81,43 @@
  *   &lt;property name="excludes" value="java.io,java.net,java.lang.Math"/&gt;
```

Comment:
in config there is 3 exceptions, but in example only 2 of them is shown. 
I think `java.net` can be excluded from config example and its description above

Reply:
Done